templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: v1
```

Comment:
Yes, I'd like to test this template a bit more then modify the PR to move the ansible-service-broker-all.yaml and use this in it's place.